### ISSUE TITLE
Local CLI not working in 1.11.1

### ISSUE THREAD

**Issue by cyrilletuzi** [cyrilletuzi/vscode-angular-schematics#26]
```
ng g component test
zsh:1: command not found: ng
```

Though changelog state that since 1.11.0 : `A global installation of Angular CLI is no longer needed, the extension will use the project's local CLI (if installed in the default directory, ie. node_modules).`

Am I missing something ?

**EDIT**

I Tried to change vscode-angular-schematics version back to 1.11.0 and it's working fine !

Output:
```
./node_modules/.bin/ng g component test
CREATE src/app/test2/test.component.scss (0 bytes)
CREATE src/app/test2/test.component.html (24 bytes)
CREATE src/app/test2/test.component.spec.ts (621 bytes)
CREATE src/app/test2/test.component.ts (266 bytes)
UPDATE src/app/app.module.ts (466 bytes)
```

(For information, I'm on macOS 10.14.2, VS Code 1.30.0, node 10.14.1, npm 6.4.1)

_Originally posted by @DDeis in https://github.com/cyrilletuzi/vscode-angular-schematics/issues/24#issuecomment-448549153_

**Comment by cyrilletuzi** [comment]
@DDeis I'll look into this soon.

As a first feedback: v1.11.1 changed what is *displayed* in the output channel (just `ng`) but it's still internally doing the command `./node_modules/.bin/ng`, *if available*.

There is 3 things to investigate:
- check if the right local command is launched (and not the global one) > probably not this, as I tested it
- check if the detection of `./node_modules/.bin/ng` is working in all environnements,
- check if it's a problem with custom shell as you're using zsh > @DDeis could you do this check by using the normal bash?

**Comment by DDeis** [comment]
Thanks for your feedback, I just tried switching VS Code "terminal.integrated.shell.osx" back to default (bash) with no success, see output:
```sh
ng g component test
/bin/bash: ng: command not found
```

**Comment by cyrilletuzi** [comment]
Fixed in 1.11.2.

@DDeis Thanks for reporting. Please take just one minute to star [the extension on Visual Studio Code marketplace](https://marketplace.visualstudio.com/items?itemName=cyrilletuzi.angular-schematics). 👍

**Comment by DDeis** [comment]
Awesome ! Thank you will do !
